datasets/ro_sts_parallel/ro_sts_parallel.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
```

Comment:
I feel like the parallel version of the Ro-STS dataset could be an additional configuration to the `ro_sts` dataset.

The `ro_sts` dataset already has a configuration named `ro_sts`. Maybe we can have an additional config named `parallel` ? What do you think ?

Reply:
We have thought about this option as well, but the data files have different structures, and different tasks attached. On the [GitHub repo](https://github.com/dumitrescustefan/RO-STS/tree/master/dataset) we have two separate folders for them, and we kind of handle them as separate datasets. (Both datasets build upon the STS dataset translation, but it is offered in two flavors, one for semantic textual similarity and the other for machine translation.) What do you think, maybe we can leave them as two different datasets?

Reply:
Ok that makes sense !